### PR TITLE
Added new uitype CurrencyInventory [IT499]

### PR DESCRIPTION
**Pull request by konadrian**
Added new uitype CurrencyInventory [IT499]

### CODE REVIEW

File: modules/Vtiger/uitypes/CurrencyInventory.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	 * {@inheritdoc}
+	 */
+	public function getDisplayValue($value, $record = false, $recordModel = false, $rawText = false, $length = false)
+	{
+		$value = parent::getDisplayValue($value);
+		$currencyId = null;
+		if ($recordModel) {
+			$currencyId = $this->getCurrencyId($recordModel);
+		}
+		if ($record && !$currencyId && $recordModel = Vtiger_Record_Model::getInstanceById($record)) {
```
Comment (TomaszQr) [inline modules/Vtiger/uitypes/CurrencyInventory.php]: ?

File: app/QueryField/CurrencyInventoryField.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+namespace App\QueryField;
+
+/**
+ * Double Query Field Class.
+ *
+ * @copyright YetiForce Sp. z o.o
+ * @license   YetiForce Public License 3.0 (licenses/LicenseEN.txt or yetiforce.com)
+ * @author    Adrian Koń <a.kon@yetiforce.com>
+ */
```
Comment (rskrzypczak) [inline app/QueryField/CurrencyInventoryField.php]: ?

File: modules/Vtiger/uitypes/CurrencyInventory.php [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+class Vtiger_CurrencyInventory_UIType extends Vtiger_Double_UIType
+{
+	/**
+	 * {@inheritdoc}
+	 */
+	public function getDisplayValue($value, $record = false, $recordModel = false, $rawText = false, $length = false)
+	{
+		$value = parent::getDisplayValue($value);
+		$currencyId = null;
+		if ($recordModel && $recordModel->getModule()->isInventory()) {
```
Comment (rskrzypczak) [inline modules/Vtiger/uitypes/CurrencyInventory.php]: ?

File: modules/Vtiger/uitypes/CurrencyInventory.php [hunk 4]
```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * UIType CurrencyInventory Field Class.
+ *
+ * @copyright YetiForce Sp. z o.o
+ * @license   YetiForce Public License 3.0 (licenses/LicenseEN.txt or yetiforce.com)
+ * @author    Adrian Koń <a.kon@yetiforce.com>
+ */
```
Comment (rskrzypczak) [inline modules/Vtiger/uitypes/CurrencyInventory.php]: ?

**Pull request merged**